### PR TITLE
♻️ limit rclone CPU and memory usage

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

- rclone is used via docker image
- when running the container limits to memory and cpu usage are enforces
- tunned down rclone usage
- fixed flaky test
- rclone no longer must be installed on the host to be used.

Bonus:
- fixes missing file removal in dynamic-sidecar
- fixes flaky tests and removed `reload` of patched modules
- removed `logging.` to log messages, since it defaults to the default root logger, making it hard to debug.

## Related issue/s

- closes https://github.com/ITISFoundation/osparc-simcore/pull/3086
- https://github.com/ITISFoundation/osparc-issues/issues/638

### CODE REVIEW

File: packages/simcore-sdk/requirements/_base.in [hunk 1]
```diff
@@ -7,7 +7,7 @@
 --requirement ../../../packages/settings-library/requirements/_base.in
 --requirement ../../../packages/models-library/requirements/_base.in
 
-aiocache
+aiodocker
 aiofiles
```
Comment (pcrespov) [inline packages/simcore-sdk/requirements/_base.in:11]: make sure this change in simcore-sdk reqs is also applied downstream (i.e. all services including simcore-sdk).
Be careful though to only do these changes in the requirements downstream though ...  (TIP: ``make reqs upgrade=aiodocker``

File: packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone.py [hunk 2]
```diff
@@ -85,41 +111,79 @@ async def sync_local_to_s3(
...
         # we must run the command from the file's directory. See below
         # example for further details:
         #
-        # local_file_path=`/tmp/pytest-of-silenthk/pytest-80/test_sync_local_to_s30/filee3e70682-c209-4cac-a29f-6fbed82c07cd.txt`
-        # s3_path=`simcore/00000000-0000-0000-0000-000000000001/00000000-0000-0000-0000-000000000002/filee3e70682-c209-4cac-a29f-6fbed82c07cd.txt`
+        # local_file_path=`/tmp8274wr8w/workspace.zip`
+        # s3_path=`simcore/30a298ba-e0bf-11ec-96a7-02420a000029/5d204f4a-253d-4c45-96af-d7fb78bd5a79/workspace.zip`
         #
-        # rclone
+        # docker
```
Comment (pcrespov) [inline packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone.py:130]: rm comment? might instead ``log.debug( "".join(r_clone_command) ) ``

File: services/dynamic-sidecar/Dockerfile [hunk 3]
```diff
@@ -22,6 +24,19 @@ RUN set -eux && \
   # verify that the binary works
   gosu nobody true
 
+# Install docker
+ENV DOCKER_VERSION="5:20.10.16~3-0~debian-buster"
```
Comment (pcrespov) [inline services/dynamic-sidecar/Dockerfile:28]: why do you need to install docker here? would it nos suffice using ``aiodocker``? This container has already ``/var/run/docker.sock`` mount

File: packages/simcore-sdk/requirements/_base.in (hunk 1 above)
Comment (GitHK) [reply packages/simcore-sdk/requirements/_base.in:11]: Thanks for the tip.

**Review by pcrespov — APPROVED** [review]
This PR will hopefully mitigate rclone's greed for resources and avoid restarting dy-sidecar which leads to a really inconsistent state.

Pair reviewed and agreed on minor changes in coming PRs

**Comment by pcrespov** [comment]
@GitHK consider adding a name to the rclone container: see https://github.com/ITISFoundation/osparc-simcore/blob/e68e748ab9b76ab11a5294b5a839dc3e156bab6a/packages/simcore-sdk/src/simcore_sdk/node_ports_common/r_clone.py#L80

**Comment by GitHK** [comment]
rclone is no longer being used. closing this for now.

**Pull request closed without merging**